src/main/java/com/google/api/codegen/RerouteLogic.java

```diff
@@ -0,0 +1,38 @@
+/* Copyright 2016 Google Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.api.codegen;
+
+import com.google.api.client.repackaged.com.google.common.base.Strings;
+import com.google.api.tools.framework.model.Interface;
+
+public class RerouteLogic {
+  /**
+   * If rerouteToGrpcInterface is set, then looks up that interface and returns it, otherwise
+   * returns the value of defaultInterface.
+   */
+  public static Interface getTargetInterface(
```

Comment:
I think we can put this into InterfaceConfig as a static method so we do not have to create a new class.